SPT-AKI Profile Editor/Core/ProfileClasses/Profile.cs

```diff
@@ -47,19 +47,20 @@ public string[] Suits
             }
         }
 
-        [JsonProperty("weaponbuilds")]
-        public Dictionary<string, WeaponBuild> WeaponBuilds
+        [JsonProperty("userbuilds")]
+        public UserBuilds UserBuilds
         {
-            get => weaponBuilds;
+            get => userBuilds;
             set
             {
-                weaponBuilds = value;
+                userBuilds = value;
+                //OnPropertyChanged("EquipmentBuilds");
                 WeaponBuildsChanged();
```

Comment:
Is property name changed in this function?

Reply:
> Is property name changed in this function?

in the WeaponBuildsChanged function?

Reply:
> in the WeaponBuildsChanged function?

Yes, OnPropertyChanged is called here for few properties, including the old WeaponBuilds

Reply:
> > in the WeaponBuildsChanged function?
> 
> Yes, OnPropertyChanged is called here for few properties, including the old WeaponBuilds

Oh yeah, sorry. I just updated it.